storage/store/my_receipts_test.go

```diff
@@ -61,7 +70,7 @@ func TestMyExecutionReceiptsStorage(t *testing.T) {
 	})
 
 	t.Run("store different receipt for same block should fail", func(t *testing.T) {
-		withStore(t, func(store *store.MyExecutionReceipts, db storage.DB) {
+		withStore(t, func(store1 storage.MyExecutionReceipts, results storage.ExecutionResults, receipts storage.ExecutionReceipts, db storage.DB) {
```

Comment:
nit: maybe just rename it to `str` or something since there is no store2

Reply:
does it actually need to be renamed from `store`? I don't see a conflict in this PR